code/game/machinery/computer/launchpad_control.dm

```diff
@@ -256,8 +167,108 @@ DEFINE_BUFFER_HANDLER(/obj/machinery/computer/launchpad)
 			. = TRUE
 		if("launch")
 			teleport(usr, current_pad, TRUE)
+			if (recall_time)
+				addtimer(CALLBACK(src, PROC_REF(teleport), usr, current_pad, FALSE), recall_time * 10)
 			. = TRUE
 
 		if("pull")
 			teleport(usr, current_pad, FALSE)
 			. = TRUE
+
+/obj/machinery/computer/launchpad/Initialize(mapload)
+	launchpads = list()
+	. = ..()
+	AddComponent(/datum/component/usb_port, list(
+		/obj/item/circuit_component/bluespace_launchpad,
```

Comment:
why move all this code fro no reason?

Also, the recall_time should be included in the USB interface.

Reply:
Because it was in the middle of the launchpad code for some reason